# -*- coding: UTF-8 -*-
"""
FileName: loginwindow.py
Author: jiaminbit@sina.com
Create date: 2017.6.20
description: login UI
Update dateï¼š2017.7.20
version 1.0.0
"""

from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_login(object):
    def setupUi(self, login):
        login.setObjectName("login")
        login.resize(491, 326)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(login.sizePolicy().hasHeightForWidth())
        login.setSizePolicy(sizePolicy)
        login.setStyleSheet("")
        self.gridLayout_2 = QtWidgets.QGridLayout(login)
        self.gridLayout_2.setContentsMargins(0, 0, 0, 12)
        self.gridLayout_2.setSpacing(0)
        self.gridLayout_2.setObjectName("gridLayout_2")
        self.gridLayout = QtWidgets.QGridLayout()
        self.gridLayout.setSpacing(0)
        self.gridLayout.setObjectName("gridLayout")
        self.lb_layout = QtWidgets.QLabel(login)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.lb_layout.sizePolicy().hasHeightForWidth())
        self.lb_layout.setSizePolicy(sizePolicy)
        self.lb_layout.setText("")
        self.lb_layout.setObjectName("lb_layout")
        self.gridLayout.addWidget(self.lb_layout, 3, 3, 1, 1)
        self.le_pwd = QtWidgets.QLineEdit(login)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.le_pwd.sizePolicy().hasHeightForWidth())
        self.le_pwd.setSizePolicy(sizePolicy)
        self.le_pwd.setEchoMode(QtWidgets.QLineEdit.Password)
        self.le_pwd.setObjectName("le_pwd")
        self.gridLayout.addWidget(self.le_pwd, 2, 3, 1, 1)
        self.lb_pwd = QtWidgets.QLabel(login)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.lb_pwd.sizePolicy().hasHeightForWidth())
        self.lb_pwd.setSizePolicy(sizePolicy)
        self.lb_pwd.setObjectName("lb_pwd")
        self.gridLayout.addWidget(self.lb_pwd, 2, 1, 1, 1)
        self.cb_user = QtWidgets.QComboBox(login)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.cb_user.sizePolicy().hasHeightForWidth())
        self.cb_user.setSizePolicy(sizePolicy)
        self.cb_user.setObjectName("cb_user")
        self.cb_user.addItem("")
        self.cb_user.addItem("")
        self.gridLayout.addWidget(self.cb_user, 1, 3, 1, 1)
        self.lb_user = QtWidgets.QLabel(login)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.lb_user.sizePolicy().hasHeightForWidth())
        self.lb_user.setSizePolicy(sizePolicy)
        self.lb_user.setObjectName("lb_user")
        self.gridLayout.addWidget(self.lb_user, 1, 1, 1, 1)
        self.lb_image = QtWidgets.QLabel(login)
        self.lb_image.setText("")
        self.lb_image.setScaledContents(True)
        self.lb_image.setObjectName("lb_image")
        self.gridLayout.addWidget(self.lb_image, 1, 0, 2, 1)
        self.pb_login = QtWidgets.QPushButton(login)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.pb_login.sizePolicy().hasHeightForWidth())
        self.pb_login.setSizePolicy(sizePolicy)
        self.pb_login.setMaximumSize(QtCore.QSize(100, 200))
        self.pb_login.setLayoutDirection(QtCore.Qt.LeftToRight)
        self.pb_login.setObjectName("pb_login")
        self.gridLayout.addWidget(self.pb_login, 4, 1, 1, 1)
        self.pb_exit = QtWidgets.QPushButton(login)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.pb_exit.sizePolicy().hasHeightForWidth())
        self.pb_exit.setSizePolicy(sizePolicy)
        self.pb_exit.setMaximumSize(QtCore.QSize(100, 16777215))
        self.pb_exit.setLayoutDirection(QtCore.Qt.LeftToRight)
        self.pb_exit.setObjectName("pb_exit")
        self.gridLayout.addWidget(self.pb_exit, 4, 3, 1, 1)
        self.gridLayout_2.addLayout(self.gridLayout, 1, 0, 1, 1)
        self.lb_login = QtWidgets.QLabel(login)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Expanding)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.lb_login.sizePolicy().hasHeightForWidth())
        self.lb_login.setSizePolicy(sizePolicy)
        font = QtGui.QFont()
        font.setPointSize(25)
        font.setBold(True)
        font.setItalic(True)
        font.setWeight(75)
        self.lb_login.setFont(font)
        self.lb_login.setStyleSheet("background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));")
        self.lb_login.setObjectName("lb_login")
        self.gridLayout_2.addWidget(self.lb_login, 0, 0, 1, 1)

        self.retranslateUi(login)
        QtCore.QMetaObject.connectSlotsByName(login)

    def retranslateUi(self, login):
        _translate = QtCore.QCoreApplication.translate
        login.setWindowTitle(_translate("login", "Login"))
        self.lb_pwd.setText(_translate("login", "PassWord:"))
        self.cb_user.setItemText(0, _translate("login", "Administrator"))
        self.cb_user.setItemText(1, _translate("login", "Operator"))
        self.lb_user.setText(_translate("login", "User Name:"))
        self.pb_login.setText(_translate("login", "Login"))
        self.pb_exit.setText(_translate("login", "Exit"))
        self.lb_login.setText(_translate("login", "  Login System"))

